src/components/AccountSwitcher.tsx

```diff
@@ -73,7 +81,7 @@ function AccountSwitcher() {
             shouldShowRightIcon: true,
             iconRight: Expensicons.Checkmark,
             success: true,
-            isSelected: true,
+            isSelected: focusedAccountID === currentUserPersonalDetails.accountID,
```

Comment:
Why we are making this change? The current user should always be marked as selected